src/powershell/check-defender.ps1

```diff
@@ -0,0 +1,7 @@
+# Ensure that Windows Defender has been uninstalled
+Write-Output "[ ] Checking if Windows Defender is running"
+Get-Service -Name WinDefend -ErrorVariable err -ErrorAction SilentlyContinue
+if (!$err) {
```

Comment:
Is it worth verifying that the error here is the correct (expected) error?

Reply:
I understand and agree with what you are doing here.  My question is if it is possible/worthwhile to verify that if an error occurred, is it in fact the error that you expected (service not running) and not a different error?

Reply:
sorry for misunderstanding your request. please see https://github.com/cisagov/windows-server-packer/pull/20/files/42414076cb77db0e016d078a5aa8117fc5adbbe0..4faaebb51a23f2ceb50d541e3951ef25d2ed51f5

Reply:
The above commit looks correct to me.  I am signing off on this for @dav3r since he is away for a few days.